README.md

```diff
@@ -10,7 +10,7 @@ In this project I built a simple yet powerful webpack boilerplate, which will be
 # Getting Started
 To get a local copy up and running, follow these simple steps:
 ## Usage
-First copy this repo and clone it on your machine using this [link](git@github.com:Ibnballo1/webpack.git).
+First copy this repo and clone it on your machine using this [link](git@github.com:Ibnballo1/ToDo-List-App.git).
 Then cd into webpack.
```

Comment:
Please add `npm install`, and `npm start` commands in README so a user can install the dependencies and run the app.